src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -243,6 +243,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
     backendCall.instantiateCommand match {
       case Success(command) => runWithJes(backendCall, command, jesInputs, jesOutputs, runIdForResumption)
+      case Failure(ex: SocketTimeoutException) => throw ex // probably a GCS transient issue, throwing will cause it to be retried
```

Comment:
So, a Future.failed means "retry" but a FailedExecutionHandle means "don't retry"? Could we have a RetryMeExecutionHandle or is that too awkward?